SchoolXplorer.Application/Services/ISchoolDistrictService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SchoolXplorer.Application.Dtos;
+
+namespace SchoolXplorer.Application.Services
+{
+	public interface ISchoolDistrictService
+	{
+		Task<SchoolDistrictDto> CreateSchoolDistrictAsync(CreateSchoolDistrictDto createSchoolDistrictDto);
```

Comment:
Avoid repeating same words. 
In a service called school district if you have a method called create it’s inffered that it’s for creating school district over specifications are unnecessary

Reply:
Sure, updated method name to  CreateAsync